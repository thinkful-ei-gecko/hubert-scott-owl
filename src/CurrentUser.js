import React from 'react';

export default function CurrentUser(props) {
    return (
        <div className="current-user">
            <div className="volume-line"></div>
            <div className="user-avatar">
                <img src={props.avatar} alt="user avatar"/>
            </div>
        </div>
    );
}