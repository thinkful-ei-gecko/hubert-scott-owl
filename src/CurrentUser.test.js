import React from 'react';
import ReactDOM from 'react-dom';
import CurrentUser from './CurrentUser';
import renderer from 'react-test-renderer';

describe('<CurrentUser />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');

        ReactDOM.render(<CurrentUser  id = 'a' name = 'frank' avatar = 'https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1'/>, div)

        ReactDOM.unmountComponentAtNode(div);
    });
});

describe('<CurrentUser />', () => {
    it('renders UI correctly', () => {
    const tree = renderer.create(<CurrentUser  id = 'a' name = 'frank' avatar = 'https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1'/>).toJSON();
    expect(tree).toMatchSnapshot();
    })
})