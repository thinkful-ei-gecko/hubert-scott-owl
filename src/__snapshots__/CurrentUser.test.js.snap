// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CurrentUser /> renders UI correctly 1`] = `
<div
  className="current-user"
>
  <div
    className="volume-line"
  />
  <div
    className="user-avatar"
  >
    <img
      alt="user avatar"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
  </div>
</div>
`;
